.github/workflows/test.yml

```diff
@@ -79,24 +79,12 @@ jobs:
           echo "======"
           conda list
       # Temporarily pin panel and nbconvert
-      - name: Pin panel and nbconvert on py36
+      - name: Pin panel on py36
         if: matrix.python-version == '3.6'
         run: |
           eval "$(conda shell.bash hook)"
           conda activate test-environment
-          conda install -c pyviz "panel=0.12.1" "nbconvert>5"
-      - name: Pin panel and nbconvert on py>36
-        if: matrix.python-version != '3.6'
-        run: |
-          eval "$(conda shell.bash hook)"
-          conda activate test-environment
-          conda install -c pyviz "panel=0.12.2" "nbconvert>5"
-      - name: Patch with an older version of freetype since 2.11.0 is broken with Matplotlib on Windows
-        if: (matrix.os == 'windows-latest')
-        run: |
-          eval "$(conda shell.bash hook)"
-          conda activate test-environment
-          conda install ${{ env.CHANS_DEV}} "freetype=2.10.4"
+          conda install -c pyviz "panel=0.12"
```

Comment:
Pinning panel *should* be fine for holoviews testing (and I am happy to merge this) but long-term it would be better if we could let `panel` float to the latest version.

Reply:
Yes in this particular case I've had to pin panel there are one or more dev releases in the 0.13 series that can be installed on Python 3.6 but that are actually not working on Python 3.6.